components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_rk4.F

```diff
@@ -1645,6 +1831,11 @@ subroutine ocn_time_integrator_rk4_cleanup(block, dt, err)!{{{
 
       call ocn_vmix_implicit(dt, meshPool, statePool, forcingPool, scratchPool, err, 2)
 
+      if (config_enable_nonhydrostatic_mode .and. .not. config_enable_vertMomentum_disable_nonHydro) then
```

Comment:
here is another flag to remove -- I don't envision needing config_enable_vertMomentum_disable_nonHydro, but I'm okay with leaving it if we want it for longer term debugging.